### DISCUSSION TITLE
nominatim replicaton

### DISCUSSION THREAD

**Discussion by martimlello07** [osm-search/Nominatim#2984] [General]
hi.
i created a service for nominatim replication and its working.
my question is whether updates continue to work with more than one imported country in the db.

thank you

**Comment by mtmail** [comment]
https://nominatim.org/release-docs/develop/admin/Advanced-Installations/ covers how such a case needs to be setup.

**Reply by martimlello07** [reply]
i have runned the script to import multiple regions, and that structure of directory with sequence.state files is already created.
so after that with the nominatim replication,  will the database update?
